packages/datadog-plugin-fastify/src/fastify.js

```diff
@@ -25,6 +29,8 @@ function createWrapFastify (tracer, config) {
 
 function createOnRequest (tracer, config) {
   return function onRequest (request, reply, next) {
+    if (typeof next !== 'function') return next()
```

Comment:
this seems problematic

Reply:
I fixed it below but it seems I forgot that one.